packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -157,11 +157,57 @@ describe('masternode', () => {
     })
   })
 
+  describe('getBlockHeader', () => {
+    /**
+     * Wait for block hash to reach a certain height
+     */
+
+    async function waitForBlockHash (height: number): Promise<string> {
+      await waitForExpect(async () => {
+        const info = await client.blockchain.getBlockchainInfo()
+        expect(info.blocks).toBeGreaterThan(height)
+      })
+
+      return await client.blockchain.getBlockHash(height)
+    }
```

Comment:
Seeing that everyone is using this utility for `waitForBlockHash`, you should refactor and move it towards the top level and remove all duplicates.